mirrord/config/configuration.md

```diff
@@ -244,36 +152,38 @@ We provide sane defaults for this option, so you don't have to set up anything h
 }
 ```
 
-### agent.communication_timeout {#agent-communication_timeout}
-
-Controls how long the agent lives when there are no connections.
-
-Each connection has its own heartbeat mechanism, so even if the local application has no
-messages, the agent stays alive until there are no more heartbeat messages.
+### agent.annotations {#agent-annotations}
 
-### agent.startup_timeout {#agent-startup_timeout}
+Allows setting up custom annotations for the agent Job and Pod.
 
-Controls how long to wait for the agent to finish initialization.
+```json
+{
+  "annotations": { "cats.io/inject": "enabled" }
+}
+```
 
-If initialization takes longer than this value, mirrord exits.
+### agent.check_out_of_pods {#agent-check_out_of_pods}
 
-Defaults to `60`.
+Determine if to check whether there is room for agent job in target node. (Not applicable
```

Comment:
The phrasing here is a bit weird. Do you intend to update docs in this PR, or just update the file itself?

Reply:
Didn't do any manual edits (such edits would be overridden anyways the next time we run medschool). All I did is run medschool and create a changelog entry file. This line is not new in this PR, it just moved.

Reply:
Yeah I know. Just spotted this while looking if the output looks correct, that's why I'm asking if you want to use this as an opportunity to go over the actual docs texts and improve them, or if you just want to update the file in this PR.

Reply:
> I'm asking if you want to use this as an opportunity to go over the actual docs texts and improve them

No thank you 😄 
Just the medschool rerun